tests/helpers/advertisement.ts

```diff
@@ -0,0 +1,33 @@
+import { Advertisement } from "../../src/models";
+
+type AdvertisementTypes = {
```

Comment:
This seems wrong to me, it should be `type AdvertisementTypes = "POPUP" | "MENU" | "BANNER"`

What is meant by other properties here:- https://github.com/PalisadoesFoundation/talawa-api/blame/61d4122b769c91d70b59bda26477b35ef773eb81/src/models/Advertisement.ts#L6

@palisadoes

Reply:
Not sure. @SiddheshKukade What was your intention with `AdvertisementTypes`?

Reply:
@xoldyckk  it's just to differentiate between a popup and banner add. Start date and end date are basically when the campaign should be promoted in the app.

Reply:
@SiddheshKukade no i meant what's the point of making it an object? The differentiation between types of advertisement can be done using just a discriminated string union like I suggested right?

Reply:
Just noticed that the typescript definition and mongoose schema don't match:-

https://github.com/PalisadoesFoundation/talawa-api/blame/61d4122b769c91d70b59bda26477b35ef773eb81/src/models/Advertisement.ts#L6

https://github.com/PalisadoesFoundation/talawa-api/blame/61d4122b769c91d70b59bda26477b35ef773eb81/src/models/Advertisement.ts#L61

In typescript, `advertisement.type` is an `object` with a single field named `type`, in mongoose model `advertisement.type` is a `string`. The type definition is incorrect.

Reply:
@xoldyckk Do I need to change type in TestAdvertisementType to string as in mongoose model?

Reply:
@Vaishali054 yes

Reply:
@xoldyckk made all the changes to the codebase